src/content/docs/tutorials/map-data-to-ocsf.mdx

```diff
@@ -358,20 +355,30 @@ The Context attributes provide enhancing information. Most notably, the
 // === Context ===
 ocsf.metadata = {
   log_name: "conn.log",
-  logged_time: zeek._write_ts,
+  logged_time: move zeek.?_write_ts,
   product: {
     name: "Zeek",
     vendor_name: "Zeek",
     cpe_name: "cpe:2.3:a:zeek:zeek",
   },
-  uid: zeek.uid,
+  uid: move zeek.uid,
   version: "1.4.0",
 }
-drop zeek._path, zeek._write_ts, zeek.uid
-ocsf.app_name = zeek.service
-drop zeek.service
+drop zeek.?_path // implied in metadata.log_name
```

Comment:
Note: Depending on whether we merge https://github.com/tenzir/tenzir/pull/5128 this may need to be adapted.